experimental/sdg-tracker/src/components/search/Search.tsx

```diff
@@ -223,6 +224,8 @@ const Search = () => {
   const [variableDcids, setVariableDcids] = useState<string[]>([]);
   const [placeDcids, setPlaceDcids] = useState<string[]>([]);
   const [errorMessage, setErrorMessage] = useState("");
+  const [userMessage, setUserMessage] = useState("");
```

Comment:
nit: do we need the userMessage as a state? because we can just read the userMessage from the fulfillResponse?